src/Workspaces/Core/Desktop/Workspace/Storage/PersistentStorageService.cs

```diff
@@ -22,15 +22,12 @@ internal abstract partial class AbstractPersistentStorageService : IPersistentSt
         protected readonly IOptionService OptionService;
         private readonly SolutionSizeTracker _solutionSizeTracker;
 
-        private readonly object _lookupAccessLock;
-        private readonly Dictionary<string, AbstractPersistentStorage> _lookup;
```

Comment:
@sharwell I removed _lookup and that bring me to clean up. @jasonmalinowski pointed out _lookup is actually not needed since we have _primarySolutionStorage saved below.